controller/registries/nacos/v2/config_test.go

```diff
@@ -46,6 +50,87 @@ func TestNewClient(t *testing.T) {
 	_, err := NewClient(config)
 	assert.Nil(t, err)
 
+	patches.ApplyFunc(clients.NewNamingClient, func(param vo.NacosClientParam) (naming_client.INamingClient, error) {
+		return nil, fmt.Errorf("err")
+	})
+
+	_, err = NewClient(config)
+	assert.Error(t, err)
```

Comment:
The test failed because of the bug of gomonkey when running on mac:
```
--- FAIL: TestNewClient (0.00s)
    config_test.go:53: 
        	Error Trace:	/Users/lzx/git/htnn/controller/registries/nacos/v1/config_test.go:53
        	Error:      	An error is expected but got nil.
        	Test:       	TestNewClient
FAIL
coverage: 16.2% of statements in mosn.io/htnn/...
FAIL	mosn.io/htnn/controller/registries/nacos/v1	7.662s
--- FAIL: TestNewClient (0.00s)
    config_test.go:58: 
        	Error Trace:	/Users/lzx/git/htnn/controller/registries/nacos/v2/config_test.go:58
        	Error:      	An error is expected but got nil.
        	Test:       	TestNewClient
FAIL
coverage: 19.7% of statements in mosn.io/htnn/...
FAIL	mosn.io/htnn/controller/registries/nacos/v2	7.695s
```

Would you drop the monkey patch like https://github.com/mosn/htnn/commit/c0c4192a8e45ecc67ee741f0eec2c6c8bf72785f?